wee_slack.py

```diff
@@ -1177,11 +1181,12 @@ def rename(self):
     def set_unread_count_display(self, count):
         self.unread_count_display = count
         self.new_messages = bool(self.unread_count_display)
-        for c in range(self.unread_count_display):
-            if self.type == "im":
-                w.buffer_set(self.channel_buffer, "hotlist", "2")
-            else:
-                w.buffer_set(self.channel_buffer, "hotlist", "1")
+        if not self.muted:
```

Comment:
Can we do this as
```
if self.muted:
    return
```
and then leave the code as it is?

Reply:
Sure, changed it now.